pages/traveling.tsx

```diff
@@ -0,0 +1,12 @@
+import React from "react";
+import Introscreen from "../components/introscreen/Introscreen";
+
+function Traveling() {
+  return (
+    <div>
+      <Introscreen></Introscreen>
+    </div>
```

Comment:
Auf den ersten Blick sieht es so aus, als ob du hier nur den Introscreen zurückgibst. Wieso brauchst du hierfür eine extra Komponente, wenn du am Introscreen nichts veränderst bzw. ihm keine props mitgibst? Du könntest in deinem React Router unter "/traveling" direkt den Introscreen zurückgeben 😬

Zudem könntest du die div auch wieder durch Fragments ersetzten und den Introscreen selfclosing schreiben [<Introscreen/>]

Reply:
> Auf den ersten Blick sieht es so aus, als ob du hier nur den Introscreen zurückgibst. Wieso brauchst du hierfür eine extra Komponente, wenn du am Introscreen nichts veränderst bzw. ihm keine props mitgibst? Du könntest in deinem React Router unter "/traveling" direkt den Introscreen zurückgeben 😬

wir nutzen next/link aber keinen react router :-) 
https://nextjs.org/docs/routing/introduction#linking-between-pages
> 
> Zudem könntest du die div auch wieder durch Fragments ersetzten und den Introscreen selfclosing schreiben

Reply:
Ah okay. Das hatte ich nicht auf dem Schirm. Dann macht wohl Sinn